modules/dr-restoring-cluster-state.adoc

```diff
@@ -290,7 +290,8 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 ----
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
-
++
+.. Optional: For each lost control plane host, repeat the steps for verifying that the single member control plane has started successfully.
```

Comment:
The only steps that should be repeated are the crictl ones. Maybe worth opening low-prio bug to have this better clarified.